app/views/providers/submitted_applications/_irregular_incomes_table.html.erb

```diff
@@ -0,0 +1,24 @@
+<% if irregular_incomes.any? %>
+  <section>
+    <div class="govuk-!-padding-bottom-6"></div>
+    <section class="print-no-break">
+      <table class="govuk-table" style="margin-bottom: 15px;">
```

Comment:
these style attributes for margin and padding can be replaced with the govuk spacing override classes (see https://design-system.service.gov.uk/styles/spacing/ ) since these are responsive and will work on different sized screens